### PR TITLE
Update free-courses-hi.md with Free Server Management Course by CodeWithharry

### PR DESCRIPTION
**Pull request by sujal0320**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by sujal0320** [comment]
@eshellman kindly review and merge.
Thank you for your time

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -660,6 +661,11 @@
 * [Information And Cyber Security](https://www.youtube.com/playlist?list=PLYwpaL_SFmcArHtWmbs_vXX6soTK3WEJw) - 5 Minutes Engineering
 
 
+### Server Management
+
+* [Server Management and Website Deployment - Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiBNXUisDdSmfNbsKq407XC) - CodeWithHarry
```
Comment (eshellman) [inline courses/free-courses-hi.md:666]: use the devops category

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -284,6 +284,7 @@
 * [Complete DevOps Zero to Hero Course](https://www.youtube.com/playlist?list=PLdpzxOOAlwvIKMhk8WhzN1pYoJ1YU8Csa) - Abhishek.Veeramalla
 * [DevOps Training Tutorials For Beginners in Hindi](https://www.youtube.com/playlist?list=PLYEK_dHOjwtODYB46wFuc34muw9Gl5X5x) - Linux Wale Guruji
 * [DevOps Training Videos in Hindi](https://www.youtube.com/playlist?list=PLQbQOmlGYH3sxlq9ugoq1ipNFP7tus5Gd) - edureka! Hindi
+* [Devops Tutorial | Server Management and Website Deployment - Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiBNXUisDdSmfNbsKq407XC) - CodeWithHarry
```
Comment (eshellman) [inline courses/free-courses-hi.md:287]: Don't invent titles! Use the title given by the resource.

Comment (sujal0320) [reply courses/free-courses-hi.md]: @eshellman apologies for the wrong title. Fixed the issue, kindly check.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**